stix_shifter/stix_transmission/src/modules/guardium/guardium_delete_connector.py

```diff
@@ -0,0 +1,28 @@
+from ..base.base_delete_connector import BaseDeleteConnector
+import json
+from .....utils.error_response import ErrorResponder
+
+
+class GuardiumDeleteConnector(BaseDeleteConnector):
+    def __init__(self, api_client):
+        self.api_client = api_client
+
+    def delete_query_connection(self, search_id):
+        # Not implemented for Guardium
```

Comment:
If Guardium doesn't support query deletion, you need to just return `{'success': True}`.